src/Microsoft.Data/DataFrame.IO.cs

```diff
@@ -0,0 +1,282 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.Data
+{
+
+    public static class Utilities
```

Comment:
I named this Utilities for now, maybe it should be `IO` instead? I don't have a strong opinion either way

Reply:
I think we should leave `ReadCsv` on DataFrame itself. But we should ensure we aren't using internal/private functionality, so other libraries can performantly create DataFrames from other data sources.